site.yml

```diff
@@ -45,6 +45,14 @@
       tags: ['infra', 'rabbitmq']
   environment: "{{ env_vars|default({}) }}"
 
+- name: setup serverspec
+  hosts: all
+  roles:
+    - role: serverspec
+      tags: ['serverspec']
+      when: serverspec.enabled|bool
+  environment: "{{ env_vars|default({}) }}"
```

Comment:
@j2sol I remember now (one of the disadvantages of working really tired is you don't always remember things the next day.) One of the handlers we use wasn't set up until after rabbitmq play.